.obs/dockerfile/Dockerfile

```diff
@@ -1,23 +1,28 @@
 # SPDX-License-Identifier: Apache-2.0
 # Define the names/tags of the container
-#!BuildTag: rancher/elemental-builder-image/5.3:latest
-#!BuildTag: rancher/elemental-builder-image/5.3:%VERSION%
-#!BuildTag: rancher/elemental-builder-image/5.3:%VERSION%-%RELEASE%
+#!BuildTag: rancher/elemental-builder-image:latest
```

Comment:
Hmm, this might fail while we build two versions (like `5.3` and `5.4`) in parallel.

We should probably just drop the `:latest` tag ?!

Reply:
Done, we keep `latest` and also the slemicro version, but it resolves dynamically to 5.3 or 5.4